src/highdicom/utils.py

```diff
@@ -549,11 +576,14 @@ def map_coordinate_into_pixel_matrix(
         (first value: spacing between rows, vertical, top to bottom,
         increasing Row index) and the rows direction (second value: spacing
         between columns: horizontal, left to right, increasing Column index)
+    slice_spacing: float
+        Distance (in the coordinate defined by the Frame of Reference) between
+        neighboring slices. Default: 1
 
     Returns
     -------
-    Tuple[float, float]
-        (Column, Row) coordinate in the Total Pixel Matrix
+    Tuple[float, float, float]
+        (Column, Row, Slice) coordinate in the Total Pixel Matrix
```

Comment:
Here we should certainly elaborate on the meaning of the third `Slice` index. The current wording is misleading, since the coordinate may not lie **in** the total pixel matrix.

Reply:
Made some suggestions in 7255d20 . What do you think?